### PR TITLE
formatting: Update to improve readability

### PR DESCRIPTION
**Pull request by lucas-manuel**
No functionality changes.

- Update foundry.toml to declare verbosity and set up mainnet caching.
- Remove optimizer in normal testing to speed up compilation times during development (Recommend using optimizer profile in CI).
- Formatting across all contracts to be audited (not including tests)
- Remove `forge clean` in test commands since only changed files are compiled, its not necessary.

`develop` test time:
`make test  61.74s user 1.87s system 132% cpu 47.952 total`

Branch test time:
`make test  36.23s user 1.98s system 108% cpu 35.246 total`

### CODE REVIEW

**Review by ith-harvey — APPROVED** [review]
LGTM, I would make sure we get @grandizzy review as he has been championing most of the refactors

File: tests/forge/PositionManager.t.sol [hunk 1]
```diff
@@ -2456,7 +2456,7 @@ contract PositionManagerERC20PoolTest is PositionManagerERC20PoolHelperContract
         _positionManager.memorializePositions(memorializeParams);
 
         string memory uriString = _positionManager.tokenURI(tokenId);
-        emit log(uriString);
+        // emit log(uriString);
```
Comment (ith-harvey) [inline tests/forge/PositionManager.t.sol:2459]: Curious why this exists... maybe @MikeHathaway can opine?

Comment (MikeHathaway) [reply tests/forge/PositionManager.t.sol:2459]: There's no way to render strings within forge or verify output automatically, so the idea is to copy the log output and paste it into an SVG viewer to verify correct generation visually

File: src/base/Pool.sol [hunk 2]
```diff
@@ -38,17 +39,18 @@ abstract contract Pool is Clone, ReentrancyGuard, Multicall, IPool {
...
     /***********************/
 
-    InflatorState              internal inflatorState;
-    InterestState              internal interestState;
-    PoolBalancesState          internal poolBalances;
-    ReserveAuctionState        internal reserveAuction;
+    InflatorState       internal inflatorState;
+    InterestState       internal interestState;
+    PoolBalancesState   internal poolBalances;
+    ReserveAuctionState internal reserveAuction;
```
Comment (EdNoepel) [inline src/base/Pool.sol:48]: I found it easier to read when the alignment matched lines 47-50.

**Review by EdNoepel — COMMENTED** [review]
Happy with most of the changes.  Not a fan of reformatting `10**18` as `10 ** 18` however.  Would prefer changing them to `1e18`.

**Review by grandizzy — APPROVED** [review]
LGTM. Agree with Ed, I also find 1e18 usage nicer

**Review by EdNoepel — APPROVED** [review]
Much thanks.

**Pull request merged**